Jenkinsfile.download_logs

```
@@ -55,7 +55,9 @@ pipeline {
         stage('Create tickets') {
             steps {
                 dir ('assisted-installer-deployment') {
-                    sh "skipper run ./tools/triage/create_triage_tickets.py --jira-access-token ${JIRA_ACCESS_TOKEN} -v"
+                    withCredentials([file(credentialsId: 'prod-pull-secret', variable: 'PULL_SECRET')]) {
+                      sh "skipper run ./tools/triage/create_triage_tickets.py --jira-access-token ${JIRA_ACCESS_TOKEN} --pull-secret ${PULL_SECRET} -v"
```

```suggestion
                      sh "skipper run ./tools/triage/create_triage_tickets.py --jira-access-token ${JIRA_ACCESS_TOKEN} -v"
```

we can probably remove this as it defaults to ``os.environ.get("PULL_SECRET")`` in the code, no?

---

if not it should be possible to just do ``export PULL_SECRET`` the line before. I think there's no reason it won't work

---

In the end export was needed, and I preferred to explicitly add the command line argument

Also, the file needed to be copied in a directory that was accessible from within skipper run